corehq/apps/callcenter/tests/test_utils.py

```diff
@@ -384,7 +383,7 @@ def test_bulk_upload_usercases(self):
 
         old_user_case = CommCareCase.objects.get_case_by_external_id(TEST_DOMAIN, self.user._id, USERCASE_TYPE)
         self.assertEqual(old_user_case.owner_id, self.user.get_id)
-        self.assertEqual(2, len(old_user_case.xform_ids))
+        self.assertEqual(1, len(old_user_case.xform_ids))
```

Comment:
Good catch - I must've glazed over when looking at this - I thought it was because there wasn't a change to the user in the upload, but that was incorrect.  My change broke the test (despite what the `results['errors']` assertion says), because the user used in setup had an invalid name, so it _was_ actually necessary to change the username.  I just pushed a commit that fixes the username in the setup, switches this back to `2` and does a better check of the `results` object.